tbfparser/src/lib.rs

```diff
@@ -1,10 +1,11 @@
-use crate::util;
+pub mod util;
+
 use std::fmt;
 use std::io;
 use std::io::{Read, Seek, SeekFrom, Write};
 use std::mem;
 use std::vec;
-use util::amount_alignment_needed;
+use crate::util::amount_alignment_needed;
 
 #[repr(u16)]
 #[derive(Clone, Copy, Debug)]
```

Comment:
Is the allow dead code still required here? You are exporting this structure, so the it should not be necessary.

Reply:
No, it is not required. Commited a new change to remove the allow_dead